spec/services/atmosphere/cloud/satisfy_appliance_spec.rb

```diff
@@ -440,20 +441,20 @@
         let(:vmt1_1) { create(:virtual_machine_template, tenants: [t1])}
         let(:vmt2_1) { create(:virtual_machine_template, tenants: [t1])}
         let(:vmt2_2) { create(:virtual_machine_template, tenants: [t2])}
-        let(:wf_set_2) { create(:appliance_set, appliance_set_type: "workflow")}
+        let(:wf_set_2) { create(:appliance_set, appliance_set_type: 'workflow', user: u) }
         let(:at_with_tenant) { create(:appliance_type, visible_to: :all, virtual_machine_templates: [vmt1_1]) }
         let(:at_with_two_tenants) { create(:appliance_type, visible_to: :all, virtual_machine_templates: [vmt2_1, vmt2_2]) }
-        let(:a1_unrestricted) { create(:appliance, appliance_type: at_with_tenant, tenants: [t1, t2])}
-        let(:a1_restricted_unsatisfiable) { create(:appliance, appliance_type: at_with_tenant, tenants: [t2])}
+        let(:a1_unrestricted) { create(:appliance, appliance_set: wf_set_2, appliance_type: at_with_tenant, tenants: [t1, t2]) }
+        let(:a1_restricted_unsatisfiable) { create(:appliance, appliance_set: wf_set_2, appliance_type: at_with_tenant, tenants: [t2]) }
 
         let!(:vmt3) { create(:virtual_machine_template, tenants: [t2], managed_by_atmosphere: true)}
         let!(:shareable_at) { create(:appliance_type, visible_to: :all, shared: true, virtual_machine_templates: [vmt3]) }
-        let!(:wf_set_1) { create(:appliance_set, appliance_set_type: "workflow")}
+        let!(:wf_set_1) { create(:appliance_set, appliance_set_type: "workflow", user: u) }
```

Comment:
Line is too long. [91/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.